Gemfile

```
@@ -1,3 +1,3 @@
 source 'https://rubygems.org'
 
-gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', branch: 'v1'
+gem "bugsnag-maze-runner", "~> 9.9"
```

```suggestion
gem "bugsnag-maze-runner", "~> 9.0"
```

This will then give you the latest "9 point something".  as it is you will only get the latest "9 point 9 point something".

---

ok, will introduce somewhere in the next PRs